io_test.go

```diff
@@ -207,6 +208,98 @@ func TestEmissions(t *testing.T) {
 	DeleteShapefile(TestEmisFilename)
 }
 
+func TestOutputEquation(t *testing.T) {
+	cfg, ctmdata, pop, popIndices, mr := VarGridData()
+
+	emis := NewEmissions()
+	emis.Add(&EmisRecord{
+		PM25: E,
+		Geom: geom.Point{X: -3999, Y: -3999.},
+	}) // ground level emissions
+
+	d := &InMAP{
+		InitFuncs: []DomainManipulator{
+			cfg.RegularGrid(ctmdata, pop, popIndices, mr, emis),
+		},
+		CleanupFuncs: []DomainManipulator{
+			Output(TestOutputFilename, false, map[string]string{
```

Comment:
This is currently causing the test to fail because the Output function expects the output variables to be a variadic array of strings rather than a map. However, we want to change it to a map so the user can specify the name of their output variables. We will also want to add a check to make sure the output variables are not more than 10 characters long, because that is the maximum allowed for attribute names in shapefiles.